modules/nw-anp-audit-logging-concept.adoc

```diff
@@ -0,0 +1,136 @@
+2024-05-30T13:37:01.594Z|00029|acl_log(ovn_pinctrl0)|INFO|name="ANP:anp-tenant-log:Egress:2", verdict=pass, severity=warning, direction=from-lport: tcp,vlan_tci=0x0000,dl_src=0a:58:0a:80:02:11,dl_dst=0a:58:0a:80:02:14,nw_src=10.128.2.17,nw_dst=10.128.2.20,nw_tos=0,nw_ecn=0,nw_ttl=64,nw_frag=no,tp_src=37832,tp_dst=8080,tcp_flags=fin|ack
+2024-05-30T13:37:01.594Z|00030|acl_log(ovn_pinctrl0)|INFO|name="ANP:anp-tenant-log:Egress:2", verdict=pass, severity=warning, direction=from-lport: tcp,vlan_tci=0x0000,dl_src=0a:58:0a:80:02:11,dl_dst=0a:58:0a:80:02:14,nw_src=10.128.2.17,nw_dst=10.128.2.20,nw_tos=0,nw_ecn=0,nw_ttl=64,nw_frag=no,tp_src=37832,tp_dst=8080,tcp_flags=ack
+2024-05-30T13:37:37.100Z|00031|acl_log(ovn_pinctrl0)|INFO|name="ANP:anp-tenant-log:Ingress:2", verdict=pass, severity=warning, direction=to-lport: tcp,vlan_tci=0x0000,dl_src=0a:58:0a:80:02:14,dl_dst=0a:58:0a:80:02:11,nw_src=10.128.2.20,nw_dst=10.128.2.17,nw_tos=0,nw_ecn=0,nw_ttl=64,nw_frag=no,tp_src=46142,tp_dst=8080,tcp_flags=syn
+2024-05-30T13:37:37.101Z|00032|acl_log(ovn_pinctrl0)|INFO|name="ANP:anp-tenant-log:Ingress:2", verdict=pass, severity=warning, direction=to-lport: tcp,vlan_tci=0x0000,dl_src=0a:58:0a:80:02:14,dl_dst=0a:58:0a:80:02:11,nw_src=10.128.2.20,nw_dst=10.128.2.17,nw_tos=0,nw_ecn=0,nw_ttl=64,nw_frag=no,tp_src=46142,tp_dst=8080,tcp_flags=ack
+2024-05-30T13:37:37.101Z|00033|acl_log(ovn_pinctrl0)|INFO|name="ANP:anp-tenant-log:Ingress:2", verdict=pass, severity=warning, direction=to-lport: tcp,vlan_tci=0x0000,dl_src=0a:58:0a:80:02:14,dl_dst=0a:58:0a:80:02:11,nw_src=10.128.2.20,nw_dst=10.128.2.17,nw_tos=0,nw_ecn=0,nw_ttl=64,nw_frag=no,tp_src=46142,tp_dst=8080,tcp_flags=psh|ack
+2024-05-30T13:37:37.101Z|00034|acl_log(ovn_pinctrl0)|INFO|name="ANP:anp-tenant-log:Ingress:2", verdict=pass, severity=warning, direction=to-lport: tcp,vlan_tci=0x0000,dl_src=0a:58:0a:80:02:14,dl_dst=0a:58:0a:80:02:11,nw_src=10.128.2.20,nw_dst=10.128.2.17,nw_tos=0,nw_ecn=0,nw_ttl=64,nw_frag=no,tp_src=46142,tp_dst=8080,tcp_flags=ack
+2024-05-30T13:37:37.101Z|00035|acl_log(ovn_pinctrl0)|INFO|name="ANP:anp-tenant-log:Ingress:2", verdict=pass, severity=warning, direction=to-lport: tcp,vlan_tci=0x0000,dl_src=0a:58:0a:80:02:14,dl_dst=0a:58:0a:80:02:11,nw_src=10.128.2.20,nw_dst=10.128.2.17,nw_tos=0,nw_ecn=0,nw_ttl=64,nw_frag=no,tp_src=46142,tp_dst=8080,tcp_flags=fin|ack
+2024-05-30T13:37:37.101Z|00036|acl_log(ovn_pinctrl0)|INFO|name="ANP:anp-tenant-log:Ingress:2", verdict=pass, severity=warning, direction=to-lport: tcp,vlan_tci=0x0000,dl_src=0a:58:0a:80:02:14,dl_dst=0a:58:0a:80:02:11,nw_src=10.128.2.20,nw_dst=10.128.2.17,nw_tos=0,nw_ecn=0,nw_ttl=64,nw_frag=no,tp_src=46142,tp_dst=8080,tcp_flags=ack
+----
+====
+
+The following table describes ANP annotation values:
```

Comment:
Suggestion: are you expecting there to be more values added to this table over time? If not, it might be simpler to remove the table and just say "the following values are supported for the `k8s.ovn.org/acl-logging` annotation" or something similar.